doc/source/index.rst

```diff
@@ -23,6 +23,11 @@ The launcher
     :undoc-members:
     :show-inheritance:
 
+It is possible to disable Posttroll Nameserver usage for the incoming
+messages by starting ``satpy_launcher.py`` with command-line arguments
+``-n False -a tcp://<host>:<port>`` where the host and port point to a
+message publisher. Multiple publisher addresses can be given by supplying
+them with additional ``-a`` switches.
```

Comment:
Just a question here: Is this the same interface we use in other pytroll modules?

Reply:
Internally, yes. All the packages using `create_subscriber_from_dict_config()` or `create_publisher_from_dict_config()` are using the same settings, explained here: https://github.com/pytroll/posttroll/blob/main/posttroll/subscriber.py#L437-L438

Reply:
ok! but I was actually wondering if the external/command line interface was the same?

Reply:
For the incoming messages (subscribers) I found these:
- `geographic_gatherer.py` uses `-n false` and `-i tcp://...`, where `-i` is "inbound connection" and means the same as `-a` does here
- `segment_gatherer.py` reads these settings from the config file (using `nameservers` and `addresses` option names)

Internally the both use `posttroll.subscriber.create_subscriber_from_dict_config()`, so given the same combination the `nameserver` handling will be the same. The names used by Posttroll are `addresses` and `nameserver`.

The `-a` and `-n` options were present in `satpy_launcher.py` already before this PR.

Reply:
ok, we'll keep them for now then